### PR TITLE
124 initiate request button

### PR DESCRIPTION
**Pull request by summer-cook**
# summary
- initiate a request button on the `/requests` page leads to the form for the default (make-a-request) ware. requests can now be made to the default ware successfully
- uses the default ware slug to query for the default ware ID
- slug is currently set to make-a-request

# video
https://share.getcloudapp.com/GGuJdm8v

# acceptance criteria
- [ ] as a user, i can use the initiate a request button on the `/requests` page to make a request to the 'make-a-request` ware.

# notes 
should default ware slug be a constant?

### CODE REVIEW

File: pages/requests/index.js [hunk 1]
```diff
@@ -8,17 +8,26 @@ import {
...
+} from '../../utils'
 
 // TODO(alishaevn): authenticate the user
 const Requests = ({ ...props }) => {
   const router = useRouter()
   const { data: session } = useSession()
-  const { requests, isLoading, isError } = useAllRequests()
+  const { requests, isLoadingAllRequests, isAllRequestsError } = useAllRequests()
   const { user, userError, userLoading } = props
+  const { defaultWareID, isLoadingDefaultWare, isDefaultWareError } = useDefaultWare('make-a-request')
```
Comment (alishaevn) [inline pages/requests/index.js:24]: the slug will always be `make-a-request` right? so we could hard code that in the `useDefaultWare` function.

**Review by alishaevn — CHANGES_REQUESTED** [review]

File: pages/requests/index.js (hunk 1 above)
Comment (summer-cook) [reply pages/requests/index.js:24]: will it for sure always be make-a-request? or do suppliers ever call it something different?

Comment (summer-cook) [reply pages/requests/index.js:24]: ill just hard code it for now and i guess its easy enough to change if the slug varies at all by supplier

Comment (alishaevn) [reply pages/requests/index.js]: I believe that the default ware is actually scientist controlled. not something that each supplier sets up. so I'd imagine that it's always the same. 
but agreed that we can change it if not.

**Review by alishaevn — APPROVED** [review]

**Pull request merged**